src/cocotb_tools/runner.py

```diff
@@ -88,7 +89,7 @@ def _as_tcl_value(value: str) -> str:
 _sv_escape_translate_table = str.maketrans(_sv_escapes)
 
 
-def _as_sv_literal(value: str) -> str:
+def _as_sv_literal(value: Any) -> str:
```

Comment:
In general use `object` instead of `Any` unless you get complaints.
https://mypy.readthedocs.io/en/latest/dynamic_typing.html#any-vs-object
```suggestion
def _as_sv_literal(value: object) -> str:
```

Reply:
Didn't know that!